radio/src/lua/lua_api.h

```diff
@@ -37,7 +37,7 @@ extern "C" {
   // Can force loading of binary (.luac) or plain-text (.lua) versions of scripts specifically, and control
   //  compilation options. See interface.cpp:luaLoadScriptFileToState() <mode> parameter description for details.
   #if !defined(LUA_COMPILER) || defined(SIMU) || defined(DEBUG)
-    #define LUA_SCRIPT_LOAD_MODE    "T"   // prefer loading .lua source file for full debug info
+    #define LUA_SCRIPT_LOAD_MODE    "bt"   // prefer loading .lua source file for full debug info
```

Comment:
Should this change be here?

Reply:
No - added for testing in simulator. Removed it now.